### PR TITLE
misc(application_showcase): add new application entries

### PR DESCRIPTION
**Pull request by adeyinkaezra123**
## Description

Added new mindsdb-powered ai applications

### CODE REVIEW

File: docs/applications_showcase.mdx [hunk 1]
```diff
@@ -11,10 +11,16 @@ This file contains links to applications that integrate MindsDB. Please append l
...
+  Author: Ezra Adeyinka
 
-  Link: link_1, link_2, ...
+  Link: https://condense-mindsdb.vercel.app/
+</Card>
+
+<Card title="NpmForecast" icon="link" href="https://github.com/adeyinkaezra123/npmforecast">
+  Author: Ezra Adeyinka
+
+  Link: https://npmforecast.streamlit.app/
```
Comment (ZoranPandovski) [inline docs/applications_showcase.mdx:23]: Let's remove this untill it works. We released new version please check

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]

File: docs/applications_showcase.mdx (hunk 1 above)
Comment (adeyinkaezra123) [reply docs/applications_showcase.mdx:23]: Everything is functioning as expected, and I've introduced additional features to predict download frequencies for multiple packages. I appreciate your patience and the support you've provided.

image

image

Comment (adeyinkaezra123) [reply docs/applications_showcase.mdx:23]: Do you mind me writing an article on this? I believe this aligns perfectly with a timeseries prediction use case

Comment (ZoranPandovski) [reply docs/applications_showcase.mdx:23]: Yes, please

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**